pandas/tests/frame/methods/test_to_csv.py

```diff
@@ -388,7 +388,7 @@ def test_to_csv_dup_cols(self, nrows):
 
     @pytest.mark.slow
     def test_to_csv_empty(self):
-        df = DataFrame(index=np.arange(10))
+        df = DataFrame(index=np.arange(10, dtype=np.int64))
```

Comment:
this is bc when we do to_csv followed by read_csv it doesnt roundtrip int32?

Reply:
Yes, read_csv can't know the dtypes of its read-in columns, so it gives dtype int64 to integer columns.